### PR TITLE
chore(ibis): implement mdl validation scripts

### PR DESCRIPTION
**Pull request by goldmedal**
# Description
Introduce a Python script to validate if a Wren MDL is valid locally.

### CODE REVIEW

File: ibis-server/tools/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Description
+
+This folder contains useful tools and scripts for debugging and validation.
+
+# Tools
+- `mdl_validation.py`: Used to validate a Wren MDL. This script attempts to select all columns in all models.
```
Comment (grieve54706) [inline ibis-server/tools/README.md:6]: Append use example may be good.

**Review by grieve54706 — APPROVED** [review]
LGTM. Thank you for providing such a helpful tool.

File: ibis-server/pyproject.toml [hunk 2]
```diff
@@ -62,6 +62,7 @@ markers = [
 [tool.ruff]
 line-length = 88
 target-version = "py311"
+exclude = ["tools/"]
```
Comment (goldmedal) [inline ibis-server/pyproject.toml:65]: I ignored the ruff for the scripts. I think it's not for production purpose. It would be more flexible.

Comment (grieve54706) [reply ibis-server/pyproject.toml:65]: Hmmm. Having it would be a plus, but it's not essential. Go ahead.

**Comment by goldmedal** [comment]
Thanks @grieve54706

**Pull request merged**